Detectors/FIT/simulation/src/Detector.cxx

```diff
@@ -74,10 +74,11 @@ void Detector::ConstructGeometry()
 
   int nCellsA = Geometry::NCellsA;
   int nCellsC = Geometry::NCellsC;
+  //std::cout<<"@@@@ mGeometry->NCells "<<nCellsA<<" "<<nCellsC<<std::endl;
 
   Geometry geometry;
   TVector3 centerMCP = geometry.centerMCP(2);
-  LOG(DEBUG) << "@@@@ mGeometry->centerMCP " << nCellsA << " " << nCellsC << centerMCP.X();
+  std::cout << "@@@@ mGeometry->centerMCP " << nCellsA << " " << nCellsC << centerMCP.X() << std::endl;
```

Comment:
please leave this as is. We do printouts via the FairLogger mechanism. In order to see the messages simply use:
`o2sim --logseverity DEBUG2`.